packages/service-integration/setup.py

```diff
@@ -82,8 +82,9 @@ def read_reqs(reqs_path: Path) -> set[str]:
     zip_safe=False,
     entry_points={
         "console_scripts": [
-            "osparc-service-integrator=service_integration.cli:app",
```

Comment:
Please keep the old name as well just in case we are using it somewhere. Add a note to deprecate it

Reply:
will do